Sources/Container+Setup.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  Swinject.Container+Setup.swift
+//  boeckler
+//
+//  Created by Emiel van der Veen on 24/11/15.
+//
+
+import Foundation
+import Swinject
+
+extension Swinject.Container {
+    class func setup() -> Container {
+        AppConfig.registerDefaults()
+        
+        let container = self.init() { c in
```

Comment:
Werkt een return hier (ipv met let) hier niet direct? Mijn Swift is wat roestig dus wellicht zit ik er naast :-)